database/migrations/1659428601528_roles.ts

```diff
@@ -0,0 +1,19 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'roles'
+
+  public async up () {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments('id')
+      table.string('role').defaultTo('user')
```

Comment:
if it defaults to `user`, then what about `admin` role?

lets not keep default on db level